tools/accuracy_checker/accuracy_checker/adapters/action_recognition.py

```diff
@@ -1,6 +1,5 @@
 """
 Copyright (c) 2019 Intel Corporation
-
```

Comment:
This seems wrong. The otherfiles do have this blank line.

Reply:
Fixed